packages/commons/src/JansPet.class.ts

```diff
@@ -0,0 +1,13 @@
+export default class JansPet {
+  id: string;
+
+  name: string;
+
+  description: string;
+
+  constructor(id: string, name: string, description: string) {
```

Comment:
you could remove the id, because it's already autogenerated.  In general you could delete the constructor because you don't use it in you'r front end. There you just create an object with values.

Reply:
you are correct. I forgot to remove that field. Thanks for the reminder